inspirehep/dojson/conferences/rules.py

```diff
@@ -61,7 +62,7 @@ def acronym(self, key, value):
             address = parse_conference_address(raw_address)
             self['address'].append(address)
 
-    return value.get('e')
+    return force_list(value.get('e'))
```

Comment:
Not the most elegant way, but it's pointless to refactor further: this might get rewritten in a couple of schema versions.